escuro_johndeniel/Countries/script.js

```diff
@@ -0,0 +1,33 @@
+document.getElementById("search-button").addEventListener("click", function () {
```

Comment:
If you're done changing IDs in your country.html, do not forget to change also your ID naming in country.js, so that it will get the appropriate id. For example:
❌ document.getElementById("search-button")
✅ document.getElementById("search_button")

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

Reply:
Also in line 1, 2 & 7.